account/migrations/0003_passwordexpiry_passwordhistory.py

```diff
@@ -28,7 +29,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('password', models.CharField(max_length=255)),
-                ('timestamp', models.DateTimeField(auto_now=True)),
+                ('timestamp', models.DateTimeField(default=django.utils.timezone.now)),
```

Comment:
`auto_now=True` does not work for timestamp field because we need to set the timestamp explicitly when creating PasswordHistory via management command.